server/src/moodRouter.mjs

```diff
@@ -37,6 +38,17 @@ moodRouter.get("/latest", async (request, response) => {
   console.log(request.user.sub);
 });
 
+// function called getQuote (take the mood and return a string)
+const url = "https://zenquotes.io/api/random";
+async function getQuote() {
+  const response = await fetch(url);
+  let data = await response.json();
+  console.log(data[0].h);
+  return data[0].h;
+}
+
+getQuote(url);
```

Comment:
Here your getQuote fxn is calling a parameter, but when you defined your fxn on line 43, it doesn't take in a parameter. 

The reason why getQuote() is working is because on line 44, it's calling the url from line 42 regardless if there is a parameter passed into getQuote() or not

Reply:
Thank you for the clarification! I removed the parameter passed and it functions the same due to line 44.